script.js

```diff
@@ -257,7 +393,7 @@ function updateMatrix()
 
 function checkGameOver()
```

Comment:
Why did I repeat this function definition twice? Strange. I think you can remove it.